### DISCUSSION TITLE
Resume problem with poweroff and synced players

### DISCUSSION THREAD

**Discussion by DerDominik** [LMS-Community/slimserver#605] [General]
Usecase:
I'm want to use x RPis (with piCorePlayer) at my home, which are in a synced group and I want to directly powering off the clients if they are not required anymore. After switch on they should sync to the active stream or resume the last playlist.

Problem:
I tested with two RPis and can also simulate with two PCs and SqueezePlay.
Start Player 1 -> Start track in LMS -> Start Player 2 -> Synced
Switch off Player 1 -> Player 2 keeps playing
Switch off Player 2 -> Player 1 keeps playing --> everythings's fine
Switch off Player 1 -> Switch off Player 2 -> Switch on Player 2 -> resuming the last played track (rewinds also x seconds, fine)
Switch off Player 1 -> Switch off Player 2 -> Switch on Player 1 -> doesn't resume the last played track

Is this problem known or does noone needed this in the past?
Nice would be a setting, how LMS has to handle this (automatic resume of last playlist, wait for new start, ...)
Maybe also some more logs from LMS are required?

**Comment by mherger** [comment]
When you sync a player to another, the "other" would be the master. The first player would not change its playlist in the background, but would just play whatever was in the other's queue. Therefore switching the other player on would restore it in the previous state, whereas for the first player this is not the case.

Did you look into using the Groups plugin? It might give you more control.

**Reply by DerDominik** [reply]
That was a good hint :-)
With this, it no longer makes a difference which client is switched off last and which is switched on first.

But it seems that the stream runs permanently to the end afterwards. Is there any way to pause the stream here?
LMS notices when the last active client is switched off.
